lib/exercism/code_analyzer.rb

```diff
@@ -0,0 +1,39 @@
+class CodeAnalyzer
+  attr_reader :language, :code, :commit_id
+
+  def initialize(options = {})
+    @language = options[:language]
+    @code = options[:code]
+    @commit_id =  options[:commit_id]
+  end
+
+  def run
+    '---common---'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.